anycloud/cli/Cargo.toml

```diff
@@ -5,7 +5,7 @@ license = "Apache-2.0"
 homepage = "https://github.com/alantech/anycloud"
 documentation = "https://alantech.gitbook.io/anycloud"
 repository = "https://github.com/alantech/anycloud"
-version = "0.1.37-beta8"
+version = "0.1.37-beta26"
 authors = ["Luis de Pombo <luis@alantechnologies.com>", "David Ellis <david@alantechnologies.com>"]
```

Comment:
@cdmistman and @aguillenv you guys should add yourselves to this (and the avm Cargo.toml) authors array. :)